src/main/scala/com/cognite/sdk/scala/common/ReadWritableResource.scala

```diff
@@ -13,6 +13,15 @@ trait DeleteByIds[F[_], PrimitiveId] {
   def deleteById(id: PrimitiveId): F[Unit] = deleteByIds(Seq(id))
 }
 
+trait DeleteByIdsWithIgnoreUnknownIds[F[_], PrimitiveId] extends DeleteByIds[F, PrimitiveId] {
+  def deleteByIds(ids: Seq[PrimitiveId], ignoreUnknownIds: Boolean = false): F[Unit]
+
+  def deleteById(id: PrimitiveId, ignoreUnknownIds: Boolean = false): F[Unit] =
+    deleteByIds(Seq(id), ignoreUnknownIds)
+}
+
+final case class DeleteRequestWithIgnoreUnkownIds(items: Seq[CogniteId], ignoreUnknownIds: Boolean)
```

Comment:
typo: "Unkown" -> "Unknown"

Reply:
We should make this a generic "ItemsWithIgnoreUnknownIds" instead. There's nothing in the class that makes it specific to deletion. Move it next to the `Items` definition.

Reply:
And then we can give it a default value of `true` as well.